### PR TITLE
Docs/Config fixes

### PR DESCRIPTION
**Pull request by sirlindqvist**
Too many people use 'password' as db user password and then ask in #help for support :)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FrostTheFox** [comment]
I'm going to tag on a small little fix for the config file to this just so we don't have two super small PRs.

**Comment by FrostTheFox** [comment]
Sorry to kinda hijack your PR, just adding a few things I need to update with it :P

File: docs/extras/mysql.md [hunk 1]
```diff
@@ -44,6 +44,8 @@
    - **Error: "ERROR 1007 (HY000): Can't create database 'rocketmapdb'; database exists"**
      The rocketmapdb database already exists.
      If you're trying to start a fresh database you'll need to execute `DROP DATABASE rocketmapdb`, and then run `CREATE DATABASE rocketgomapdb`. If you want to keep the rocketmapdb but start a new one, change the name.
+   - **(1045, u"Access denied for user 'rocketmapuser'@'localhost' (using password: YES)")**
+     You are using **password** as your password for the database user **rocketmapuser**. Simply run `CREATE USER 'rocketmapuser'@'localhost' IDENTIFIED BY 'password';` and replace `'password'` to the password you want to use.
```
Comment (sebastienvercammen) [inline docs/extras/mysql.md]: Not

> are

but

> might be

And what's the effect of running `CREATE USER` again on a user that already exists? `SET PASSWORD` is built to change passwords for existing users.

**Comment by FrostTheFox** [comment]
@sebastienvercammen According to https://dev.mysql.com/doc/refman/5.7/en/set-password.html, `ALTER USER` is the preferred method for setting account passwords, but regardless, you're right it is redundant to run a CREATE USER when we've already created it. Changing now

**Pull request merged**